src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -890,81 +929,81 @@ public Money GetMatureBalance(List<StakeTx> stakeTxes)
         /// and it also has to be mature and meet requirement for minimal value.
         /// </para>
         /// </summary>
-        /// <param name="stakeTxes">List of coins that are candidates for being used for staking.</param>
+        /// <param name="utxoStakeDescriptions">List of UTXO descriptions that are candidates for being used for staking.</param>
         /// <param name="spendTime">Timestamp of the coinstake transaction.</param>
         /// <param name="maxValue">Maximal amount of coins that can be used for staking.</param>
         /// <returns>List of coins that meet the requirements.</returns>
```

Comment:
if you change elsewhere from coin to UTXO description, make sure you change it everywhere 
i haven't checked all the comments, so try to go through them and see if we are somewhere still using `coin` for this UtxoStakeDescription

Reply:
This has not been changed on purpose because of issue #754. When I get to that issues I'll check what was intended by the writer of that piece and if there is a problem with implementation of this functionality.

Reply:
are we talking about the same thing?
i was talking about here we previous had `List of coins that are candidates` for `stakeTxes` parameter, we changed that to `UTXO descriptions` 

but in returns tag, we still return `List of coins`, which is inconsistent

also i've just noticed in StakeAndSignBlockAsync we have 
` /// <param name="utxoStakeDescriptions">List of coins that are available in the wallet for staking.</param>`

which is also inconsistent - so i was talking about going through the all comments and make this consistent